features/FEATURE_LWIP/lwip-interface/CONTRIBUTING.md

```diff
@@ -0,0 +1,129 @@
+# Description
+
+This document describes how to update mbed OS lwip stack. Mbed OS lwip stack is a copy of
+lwip master repository. Stack is located in `FEATURE_LWIP/lwip-interface/lwip` directory.
+
+When new releases or single commits are added from lwip master repository they need to be
+fetched via mbed OS lwip fork repository. Repository is used to rename lwip source files 
+with `lwip_` prefix to make then compatible with mbed OS build system.
+
+* Lwip master repository is under lwip project (<https://git.savannah.nongnu.org/git/lwip.git>).
+
+* Mbed OS lwip fork repository is under ARMmbed (<https://github.com/ARMmbed/lwip>)
+
+## How to integrate a release from lwip master repository to mbed OS
+
+Integrating a new release is done in following steps.
+
+* Fetch the release to mbed OS lwip fork.
+
+* Prefix files in mbed OS lwip fork with `lwip_` prefix.
+
+* Merge release to mbed OS repository.
+
+### Fetching release to mbed OS lwip fork and prefixing files
+
+1. Clone mbed OS lwip fork repository.
+
+    `git clone git@github.com:ARMmbed/lwip.git`
+
+2. Go to mbed OS lwip fork root directory and add lwip master repository as remote.
+
+    `git remote add <remote name> https://git.savannah.nongnu.org/git/lwip.git`
+
+    `e.g. git remote add lwip https://git.savannah.nongnu.org/git/lwip.git`
+
+3. Pull needed release from lwip master repository using release tag. Tags are found from <https://savannah.nongnu.org/projects/lwip>.
+
+    `git pull <remote name> <tag>`
+
+    `e.g. git pull lwip STABLE-2_0_2_RELEASE_VER`
+
+4. Push tag to mbed OS lwip fork repository to keep fork in sync.
+
+    `git push origin <tag>`
+
+    `e.g. git push origin STABLE-2_0_2_RELEASE_VER`
+
+5. Create branch for release. Use same naming convention as previous branches.
+
+    `git branch <branch name> <tag>`
+
+    `e.g. git branch mbed-os-lwip-stable-2_0_2 STABLE-2_0_2_RELEASE_VER`
+
+5. Push branch to mbed OS lwip fork repository to keep fork in sync.
+
+    `git push origin <branch name>`
+
+    `e.g. git push origin mbed-os-lwip-stable-2_0_2`
+
+6. Fetch previous prefixed branch from mbed OS lwip fork repository.
+
+    `git fetch <remote name> <remote branch>:<local branch>` 
+
+    `e.g. git fetch origin mbed-os-lwip-stable-2_0_1-prefixed:mbed-os-lwip-stable-2_0_1-prefixed`
+
+7. Checkout latest prefixed branch.
+
+    `git checkout <prefixed branch>`
+
+    `e.g. git checkout mbed-os-lwip-stable-2_0_1-prefixed`
+
+8. Merge new branch to prefixed branch.
+
+    `git merge <branch name>`
+
+    `e.g. git merge mbed-os-lwip-stable-2_0_2`
+
+9. Check if there are any new or colliding files.
+
+    If there are new c-files rename those with `lwip_` prefix and make a commit about 
+    changes to prefixed branch. Also check that lwip stack header file names does not 
```

Comment:
- Please change "prefixed branch" to "the prefixed branch".
- Please add a comma after "Also".
- Please capitalize "LWIP" or put it in code format.
- Please change "does" to "do".